tests/test_other.py

```diff
@@ -11179,3 +11179,11 @@ def test_wasmfs_readfile(self):
   def test_hello_world_above_2gb(self):
     self.run_process([EMCC, test_file('hello_world.c'), '-sGLOBAL_BASE=2147483648', '-sINITIAL_MEMORY=3GB'])
     self.assertContained('hello, world!', self.run_js('a.out.js'))
+
+  def test_hello_function(self):
```

Comment:
```suggestion
  def test_docs_hello_function(self):
```

Reply:
Maybe a convention of calling tests that keep docs valid `test_doc_` makes sense? Just an idea.

Reply:
I'm not sure... maybe if we had more than one example of this.   My inclination is to instead move them into and `examples/` directory outside of tests completely.